introduction/setting-up-eslint/README.md

```diff
@@ -0,0 +1,27 @@
+In this lesson, we will learn to setup ESLint extension for Visual Studio Code.
+
+Visual Studio Code provides a lot of inbuilt features to improve the speed at which we write code. Additionally, we can install extensions for Visual Studio Code that can help with code quality and readability.
+
+The ESLint Visual Studio Code extension, which we will use in this lesson, will also help with other JavaScript related projects.
+
+## Code Quality using ESLint
+
+Code quality is an important aspect, whenever we write code in any programming language. Writing quality code helps you limit issues and errors in your application.
+
+Identifying and fixing issues, that you might have missed while coding, can be streamlined with the help of Code Quality extensions like `ESLint`.
```

Comment:
We can identify and fix issues that we might have missed while coding with the help of tools like ESLint.